contracts/eden/tests/run-start-election.cpp

```diff
@@ -0,0 +1,16 @@
+#include <nodeos-runner.hpp>
+
+TEST_CASE("Setup Eden chain with start of full election")
+{
+   nodeos_runner r("chain-start-election");
+
+   r.tester.genesis();
+   r.tester.run_election(true, 10000, true);
+   r.checkpoint("small_election");
+   r.tester.induct_n(100);
+   r.checkpoint("inductions");
+   r.tester.eden_gm.act<actions::electsettime>(s2t("2021-11-16T18:45:00.000"));
+   r.tester.start_election(true, 10000);
```

Comment:
suggestion: we can centralize the elections scenario in the current `run-full-election.wasm` file and use checkpoints to stop the runner where we want. For instance, as opposed of creating this file, just add this `start_election` to the bottom of the full-election one with a checkpoint:

```cpp
   r.tester.genesis();
   r.tester.run_election(true, 10000, true);
   r.checkpoint("small_election");
   r.tester.induct_n(100);
   r.checkpoint("inductions");
   r.tester.eden_gm.act<actions::electsettime>(s2t("2021-11-16T18:45:00.000"));
   r.checkpoint("full_election1");
   r.tester.start_election(true, 10000);
```

If someone wants to avoid the start of the new election they can just do `cltester run-full-elections.wasm -c full_election1`

I also think since we are using checkpoints we should rename the file from `run-full-elections` to `run-elections`, then it's more generic about elections scenarios....

Reply:
Oh, cool! That’s awesome!